py/DREAM/Settings/Equations/RunawayElectrons.py

```diff
@@ -61,6 +67,13 @@ def setDreicer(self, dreicer):
         """
         self.dreicer = dreicer
 
+    def setCompton(self, compton):
+        """
+        Specifies which model to use for calculating the
+        compton runaway rate.
+        """
+        self.compton = compton
+
```

Comment:
Down the line we should be able to set the photon flux here. 	COMPTON_RATE_ITER_DMS could still be used, as a shortcut for 1e18.